src/components/CommentSection.jsx

```diff
@@ -1,22 +1,39 @@
-import { useEffect } from "react"
+import { useContext, useEffect } from "react"
 import { useState } from "react"
-import { getReviewComments } from "./api"
+import { getReviewComments, postComment } from "./api"
 import { useParams } from "react-router-dom"
 import CommentCard from "./CommentCard"
 import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from "@mui/material"
+import { UserContext } from "../contexts/user"
 
 
 
 function CommentSection (){
+    const [commentInput, setCommentInput] = useState("")
     const {reviewID} = useParams()
     const [reviewComments, setReviewComments] = useState([])
     const [isLoading, setIsLoading] = useState(true)
     const [showComments, setShowcomments] = useState(false)
-
+    const { username } = useContext(UserContext)
+    const [commentErr, setCommentErr] = useState(null)
+    
     const commentHandler = (event) => {
         event.preventDefault()
         setShowcomments(!showComments)
     }
+    const postCommentHandler = (event) => {
+        event.preventDefault()
+        if(commentInput.length > 10){
+            postComment(reviewID,{username: username.username, body: commentInput})
+            .then((newComment)=>{
+                setReviewComments((currentComments)=>{
+                    return[...currentComments, newComment]
+                })
+            })
+        } else {
+            setCommentErr("Comment must be atleast 10 characters")
+        }
+    }
 
     useEffect(()=>{
         setIsLoading(true)
```

Comment:
When I write a comment and click submit the following screenshot is my view:
![image](https://user-images.githubusercontent.com/61753159/233379670-6380211b-c775-4260-8937-77dbe4c776af.png)

- My input box does not reset
- I do not see my new comment at the top of the screen or any feedback to say it's been successfully posted
- An old error message (just before this i tried to post a blank comment) does not clear.

Please look through and make sure all these things are addressed. The input box should reset after posting, the error message should disappear and the comments should update OR there should be some kind of visual feedback about success.